mirage/scenarios/registrations.ts

```diff
@@ -54,6 +54,15 @@ export function registrationScenario(
 
     server.create('registration', { id: 'beefs' });
 
+    server.create('resource', {
+        pid: '2',
```

Comment:
This PID is not valid. It should be a DOI.

Reply:
Understood, removing now.